go.mod

```diff
@@ -2,9 +2,11 @@ module github.com/talos-systems/talos
 
 go 1.13
 
-replace github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
-
-replace github.com/talos-systems/talos/pkg/machinery => ./pkg/machinery
+replace (
+	github.com/Azure/go-autorest v10.8.1+incompatible => github.com/Azure/go-autorest/autorest v0.9.1
+	github.com/docker/distribution v2.7.1+incompatible => github.com/docker/distribution v2.7.1-0.20190205005809-0d3efadf0154+incompatible
+	github.com/talos-systems/talos/pkg/machinery => ./pkg/machinery
```

Comment:
Will this cause issues when importing Talos?

Reply:
`replace` directives are ignored when using module, but good question what will happen if you import whole module, this might need similar replace

Reply:
An alternative is that we get rid of this module? Didn't we find another workaround?

Reply:
I'll look into that